bhavyamehta4439.json

```
@@ -0,0 +1,21 @@
+{
+  "name": "Bhavya Mehta",
+  "type": "personal",
+  "bio": "Pursuing Btech in CSE,
+       I am creative and cultural
+          with multiple talents."
+  "links": [
+    {
+      "name": "Follow me on Linekdin",
+      "url": "https://www.linkedin.com/in/bhavya-mehta-b6a824226",
+      "icon": "FaLinekdin"
+     }
+    ]
+    "links": [
+    {
```

corrections are needed for the json to pass:
```suggestion
  "bio": "Pursuing Btech in CSE I am creative and cultural with multiple talents",
	"links": [{
			"name": "Follow me on Linekdin",
			"url": "https://www.linkedin.com/in/bhavya-mehta-b6a824226",
			"icon": "FaLinkedin"
		},
		{
```